features/counter-style.yml.dist

```diff
@@ -21,7 +32,9 @@ compat_features:
   - api.CSSCounterStyleRule.pad
   - api.CSSCounterStyleRule.prefix
   - api.CSSCounterStyleRule.range
+  - api.CSSCounterStyleRule.speakAs
   - api.CSSCounterStyleRule.suffix
+  - api.CSSCounterStyleRule.symbols
```

Comment:
This is by far the weirdest bit. I don't know if we should be troubled by properties for unsupported descriptors, but it's at least true (it made me stop to check that these are indeed on `CSSCounterStyleRule.prototype`).

Reply:
It's surprising that this has been exposed in CSSOM in both Chrome and Firefox if they don't actually work. But if that's true, I think they ought to be `partial_implementation` in BCD. I think it'd be best to not include the keys until this is investigated.

Reply:
I looked at this more closely.

For `symbols`, the CSSOM story is accurate: the CSSOM exposure works in Firefox and Chrome, but the descriptor is only partially implemented. Implementers other than Safari don't support `<image>` values, and this seems to be correct.

For `speak-as`, I've sent https://github.com/mdn/browser-compat-data/pull/23857.

We can wait on the latter PR, or I can comment out the key until it lands. Do you have a preference (and maybe we should turn that preference into a policy)?

Reply:
I think waiting for BCD slows us down and we need a policy that allows us to think about these things once. And the risk to minimize is the risk that whatever we assume will happen doesn't happen, and we never notice we got it wrong.

I think these won't be Baseline anyway, so OK to just proceed,